grammar.js

```diff
@@ -442,6 +443,7 @@ module.exports = grammar({
       ),
     _case_clause_tuple_access: ($) =>
       seq(field("tuple", $.identifier), ".", field("index", $.integer)),
+    let_assert: ($) => seq("let assert", $._assignment),
```

Comment:
You may want to break this into two separate nodes:

```js
let_assert: ($) => seq("let", "assert", $._assignment),
```

Doing that allows tree-sitter "extras" to be inserted between the nodes, so you could have whitespace or comments between the `let` and `assert` tokens

Reply:
not sure I would want to see that between those two though.
reason being is that in future assert alone will do something else than let assert, so:

```gleam
let 
// some comment
assert Ok(a) = Ok(1) // returns Ok(a) or crashes, "I think"
```

```gleam
assert Ok(a) = Ok(1) // returns true or crashes
```

Reply:
But I am not certain.

Reply:
```gleam
    let
    // foo cool
    assert
    // foo cool
    Ok(string) =
      do_from_utf_codepoints_impl(utf_codepoints, bit_string.from_string(""))
      |> bit_string.to_string
    string

```
compiles though... the rest is the job of the formatter.

Reply:
I have added the alt impl, the tests seem to pass so @the-mikedavis and @J3RN whatever you prefer. I think mike is correct here, not tree sitters decision if gleam allows the split.